profiles/README.md

```diff
@@ -30,7 +30,7 @@ This repository and workflow begins after we applied cytominer-database.
 | Level 5 | Consensus Perturbation Profiles | `.csv.gz` | Yes |
 
 Importantly, we include files for _two_ different types of normalization: Whole-plate normalization, and DMSO-specific normalization.
```

Comment:
I've not read the whole thing yet, but I wonder if you should mention the existence of spherized data somewhere here.

Reply:
ah, good point